### DISCUSSION TITLE
Allow using $__rate_interval in Panel Title

### DISCUSSION THREAD

**Discussion by ignas-sedunovas** [grafana/grafana#48861] [Dashboards, Panels & Visualizations]
Would it be possible to enable using the `$__rate_interval` built-in variable in Panel Title?

For the same reason `$__interval` and `$__interval_ms` were enabled in #36990

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
